lib/instana.rb

```diff
@@ -17,6 +17,6 @@
   ::Instana::Activator.start
   ::Instana.agent.spawn_background_thread
 
-  ::Instana.logger.info "Stan is on the scene.  Starting Instana instrumentation version #{::Instana::VERSION}"
+  ::Instana.logger.info "Stan is on the scene.  Starting Instana instrumentation version #{::Instana::VERSION}" if ::Instana.agent.ready?
```

Comment:
Indepentently if the Agent is ready, I think it's interesting to log this message. 
I would keep it as before and add extra log messages if the Agent is not reached or is not ready to accept spans.

Reply:
This is already done here https://github.com/instana/ruby-sensor/pull/430/files#diff-6f675565aaede7a2ff6ec3a21c72397f191bbd0060a67dd8f41c1d8df10d9c7fR92:~:text=%40logger.debug(%22Agent%20connection%20timed%20out%20retrying%20after%2060%20seconds%22)

Reply:
So will remove this condition